### PR TITLE
Added new drawer screen - Other Countries

### PR DESCRIPTION
**Pull request by shubhamaniket**
**Added a new drawer Screen - Other Countries**

- Display confirmed,recovered & dead cases of all country affected.
- The data is displayed such that the country most affected comes in first of the list.

### CODE REVIEW

**Comment by sarthakpranesh** [comment]
Hey @shubhamaniket, thanks a lot for opening this PR.
There are a few minor changes though that I would like you to make before we get this merged. One being is that we have used `Functional` based programming style for the development of screens and for components we have used `Class` based programming. So can you shift your screen to the functional based programming style to make sure the source code is similar.

File: src/screens/OtherCountries.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+export default class OtherCountries extends React.Component{
+    state = {
+        loading : true,
+        data : []
+    }
+    componentDidMount(){
+        this.fetchapi();
+    }
+    fetchapi = () => {
```
Comment (sarthakpranesh) [inline src/screens/OtherCountries.js]: To keep proper modularity, we have an api and hooks folder in the src. It's better you shift this logic there to maintain sync throughout the source code

Comment (shubhamaniket) [reply src/screens/OtherCountries.js]: Ok will do that.

File: App.js [hunk 2]
```diff
@@ -8,11 +8,15 @@ import HomeScreen from "./src/screens/HomeScreen";
 import AboutScreen from './src/screens/AboutScreen';
 import PrecautionScreen from './src/screens/PrecautionScreen';
 import HelpScreen from './src/screens/HelpScreen';
+import OtherCountries from './src/screens/OtherCountries';
 
 const Drawer = createDrawerNavigator({
     " Home ": {
         screen: HomeScreen,
     },
+    " OtherCountries " : {
```
[the remark this answers is not in the document]
Comment (shubhamaniket) [reply App.js]: Ok will do that.

**Pull request closed without merging**